src/netex-convertor/period-ticket/periodTicketNetexGenerator.ts

```diff
@@ -230,14 +230,17 @@ const periodTicketNetexGenerator = (userPeriodTicket: PeriodTicket, operatorData
         fareTableFareFrameToUpdate.prerequisites.FareFrameRef.ref = `epd:UK:${userPeriodTicket.nocCode}:FareFrame_UK_PI_FARE_PRODUCT:${placeHolderGroupOfProductsName}@pass:op`;
         fareTableFareFrameToUpdate.PricingParameterSet.id = `op:Pass@${placeHolderGroupOfProductsName}`;
 
-        fareTableFareFrameToUpdate.fareTables.FareTable = getFareTableList(userPeriodTicket, placeHolderGroupOfProductsName);
+        fareTableFareFrameToUpdate.fareTables.FareTable = getFareTableList(
+            userPeriodTicket,
+            placeHolderGroupOfProductsName,
+        );
 
         if (isGeoZoneTicket(userPeriodTicket)) {
-            fareTableFareFrameToUpdate.fareTables.FareTable.includes.FareTable = getGeoZoneFareTable(
-                userPeriodTicket);
+            fareTableFareFrameToUpdate.fareTables.FareTable.includes.FareTable = getGeoZoneFareTable(userPeriodTicket);
         } else if (isMultiServiceTicket(userPeriodTicket)) {
             fareTableFareFrameToUpdate.fareTables.FareTable.includes.FareTable = getMultiServiceFareTable(
-                userPeriodTicket);
+                userPeriodTicket,
```

Comment:
Why has prettier put a comma after userPeriodTicket here but not above?

Reply:
It's because it's on a new line unlike the one above